shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/server/plugins/PluginManager.java

```diff
@@ -1130,6 +1155,33 @@ public static CallSite bootstrap(
         MethodHandles.foldArguments(methodHandle, instance.dynamicInvoker()));
   }
 
+  /** Bootstrap method to get a plugin collection as a stream of throttleable services */
+  public static CallSite bootstrapServices(
+      Lookup lookup, String methodName, MethodType methodType, String... fileNames) {
+    // Our goal here is to create  list of services that plugins use so we can block an olive if
+    // that happens. We're given the file names of those plugins as parameters to this method and
```

Comment:
`if that happens` what is `that` in this context?

Reply:
Reworded.